pkg/bootstrap/config.go

```diff
@@ -61,7 +61,7 @@ const (
 	requiredEnvoyStatsMatcherInclusionSuffixes = rbacEnvoyStatsMatcherInclusionSuffix + ",downstream_cx_active" // Needed for draining.
 
 	// required for metrics based on stat_prefix in virtual service.
-	requiredEnvoyStatsMatcherInclusionRegexes = "vhost.*.route.*"
+	requiredEnvoyStatsMatcherInclusionRegexes = "vhost\\.*\\.route\\.*"
```

Comment:
nit: use backtick instead of " to avoid needing double escapes

Reply:
ack, fixed it.